spacy/kb/kb.pyx

```diff
@@ -32,8 +32,8 @@ cdef class KnowledgeBase:
 
     def get_candidates_batch(self, mentions: Iterable[Span]) -> Iterable[Iterable[Candidate]]:
         """
-        Return candidate entities for specified texts. Each candidate defines the entity, the original alias,
-        and the prior probability of that alias resolving to that entity.
+        Return candidate entities for specified texts. Each candidate defines the entity, the original mention,
+        and the prior probability of this mention resolving to that entity.
```

Comment:
Same question here (and in the next method) about whether or not the candidate will actually include the alias/mention.

Reply:
Updated in https://github.com/explosion/spaCy/pull/12341/commits/3cfc1c6accb4f75b141b075aa53d8d9ae12166f0 and https://github.com/explosion/spaCy/pull/12341/commits/80fb0666b98ff0bd96c36daf5f7d0e5e00ecdb25.